website/src/components/webinars/WebinarLayout.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react'
+
+import classNames from 'classnames'
+import styles from './WebinarLayout.module.scss'
+
+import { ContentSection } from '../content/ContentSection'
+import { breakpoints } from '../../breakpoints'
+import { useWindowWidth } from '../../hooks'
+import { useHubSpot } from '../../hooks/hubSpot'
+import { useChiliPiper } from '../../hooks/chiliPiper'
+
+interface Props {
+    customer?: Customer
+    title: string
+    subtitle: string
+    description: React.ReactNode
+    speakers: Speaker[]
+    children?: React.ReactNode
+}
+
+interface Customer {
+    name: string
+    logo: string
+    href: string
+}
+
+interface Speaker {
+    name: string
+    title: string
+    img: string
+    bio: string
+}
+
+export const WebinarLayout: React.FunctionComponent<Props> = ({
+    title,
+    subtitle,
+    customer,
+    description,
+    speakers,
+    children,
+}) => {
+    useHubSpot({
+        portalId: '2762526',
+        formId: '66361163-5e08-4be3-8ab0-6590b70df69e',
+        targetId: 'form',
+    })
+    useChiliPiper()
+
+    const windowWidth = useWindowWidth()
+    const isMdScreen = windowWidth < breakpoints.lg && windowWidth > breakpoints.sm
+    const isMobile = windowWidth < breakpoints.md
+
+    return (
+        <>
+            {/* Hero style differs if webinar is customer- vs. interal-based */}
+            <section className={classNames(!customer && styles.hero)}>
```

Comment:
I really dig what you wrote here for the switch logic on the hero for customers vs. internal cases! 💫

Reply:
Thank you! :)